RetailSuiteAccounting/Operation/10_ManageReceipts.md

```diff
@@ -0,0 +1,194 @@
+
+
+## Book a receipt
+
+The *BOOK RECEIPTS* tab allows to assign manually external receipts uploaded in the system that have not been registered yet in the *Accounting* module. Suppliers' invoices can be uploaded in PDF format in the *Documents* module, which will be then recognized by the system and displayed in the *BOOK RECEIPTS* tab.
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- A receipt is uploaded in PDF format in the *Documents* module.
+
+#### Procedure
+
+*Accounting > Select fiscal year > Book > Tab BOOK RECEIPTS*
+
+![New receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/NewDocument.png "[New receipt]")
+
+  > [Info] The external receipts uploaded in the *New documents* folder in the *Documents* module are displayed here. If there are no receipts to be booked, the message *No documents left to be registered* is displayed when clicking the *BOOK RECEIPTS* tab.
+
+  ![No receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/NoDocument.png "[No receipt]")
+
+1. Select the appropriate currency in the *Currency code* field, if necessary.
+
+  > [Info] Currency exchange rate can also be adjusted if needed in the *Rate* field.
+
+2. Enter the total invoice value in the *Turnover* field.
+
+3. Enter an account number in the *Contra Account* field.
+
+4. Enter the date for the manual booking in the *Date* field. Date format must be DDMMYY or DD.MM.YY.
+
+5. Enter the creditor (supplier) account number in the *Account* field.
+
+  > [Info] Note that the fields *Contra Account*, *Date*,  *Account* must be filled out to be able to book a receipt.
+
+6. Enter any other relevant booking information in the corresponding fields.
+
+  > [Info] Even though it is not compulsory, it is highly recommended to fill in the *Receipt 1* field, as the system identifies and groups together any subsequent bookings taking this field as reference.
+
+  ![Booking a receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/NewDocument01.png "[Booking a receipt]")
+
+7. Click the [BOOK] button.  
+A message above the entry line confirms that the receipt has been linked to the booking. A new booking is displayed in the bookings list in the *BOOKING* tab. A ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon (=Beleg, EN: receipt) is displayed next to the booking in the bookings list.
+
+  ![Receipt linked](../../Assets/Screenshots/RetailSuiteAccounting/Book/BookReceipts/DocumentLinked01.png "[Receipt linked]")
+
+  > [Info] If there are further unregistered external receipts, they will be displayed in the *BOOK RECEIPTS* tab. Otherwise, the message "No documents left to be registered" is displayed.
```

Comment:
```suggestion
  > [Info] If there are further unregistered documents, they will be displayed in the *BOOK RECEIPTS* tab. Otherwise, the notice "No documents left to be registered" is displayed.
```